components/Pamphlet/Pamphlet.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {
+  Button,
+  Container,
+  Header,
+  Image,
+  List
+} from 'semantic-ui-react';
+
+import styles from './Pamphlet.less';
+
+export const Pamphlet = (props) => {
+  const { title, priceText, description, imageUrl, upgradeOptions, buttonText } = props;
+
+  return (
+    <>
+      <Container className={styles.container}>
+        <Image
+          className={styles.bannerImage}
+          src={imageUrl}
+        />
+
+        <div className={styles.containerBody}>
+          <Header
+            as='h3'
+            className={styles.title}>
+            {title}
+          </Header>
+          <p className={styles.priceText}>{priceText}</p>
+          <p className={styles.description}>{description}</p>
+
+          <div className={styles.listContainer}>
+            <List>
+              <p className={styles.listDescription}>{'⭐️ You will get:'}</p>
+
+              <ul className={styles.listItems}>
+                {upgradeOptions.map((option, i) => (<li key={i}>{option}</li>))}
```

Comment:
Another content-awareness problem. It's probably better to have the importing service pass an entire block of HTML here.

So like, `body={ <ul><li>foo</li> }`

or you can use props.children here.